judoscale-ruby/test/config_test.rb

```diff
@@ -60,6 +60,20 @@ module Judoscale
       end
     end
 
+    it "initializes the config from default Amazon ECS ENV vars" do
+      # Users must set JUDOSCALE_URL manually when using Amazon ECS.
+      env = {
+        "ECS_CONTAINER_METADATA_URI" => "http://169.254.170.2/v3/a8880ee042bc4db3ba878dce65b769b6-2750272591",
```

Comment:
Seems obvious but just want to make sure our bases are covered — the last value there _is_ globally unique to separate tasks spun up under the same service umbrella?

Reply:
As far as I can tell, yes, but I'm not 100% sure. We don't actually use it for anything publicly available, though, so it's kind of arbitrary.

Reply:
Works for me!